CONTRIBUTING.md

```diff
@@ -178,12 +173,14 @@ accepted by Free Code Camp proper.
 Free Code Camp Issue Mods and staff are on hand to assist with Pull Request
 related issues on our Help Contributors Chat Room
 
-#### Methods
+#### How to find the code in the Free Code Camp codebase to fix/edit?
 
-There are two methods of creating a Pull for Free Code Camp:
-
--   Editing files via the GitHub Interface
--   Editing files on a local clone
+The best way to find out any code, you wish to change/add or remove is using 
+the GitHub search bar at the top of the repository page. For example, you could 
+search for a challenge name and the results will display all the files along 
+with line nos. Then you can proceed to the files and verify this is the area 
+that you were looking forward to edit. Always feel free to reachout to the chat
+room when you are not certain of any thing specific in the code.
```

Comment:
Should we add in a link to the [HelpContributors](https://gitter.im/FreeCodeCamp/HelpContributors) room here?